val/src/avs_timer.c

```diff
@@ -44,6 +44,10 @@ val_timer_execute_tests(uint32_t level, uint32_t num_pe)
       }
   }
 
+  if (g_single_module != SINGLE_MODULE_SENTINEL && g_single_module != AVS_TIMER_TEST_NUM_BASE) {
+    return AVS_STATUS_SKIP;
```

Comment:
Add log print for consistency in log
val_print(AVS_PRINT_TEST, "      USER Override - Skipping all Timer tests (running only a single module)\n", 0);

Reply:
Fixed.